lib/radiator/directory/audio_publication.ex

```diff
@@ -27,10 +30,14 @@ defmodule Radiator.Directory.AudioPublication do
     audio_publication
     |> cast(attrs, [
       :publish_state,
-      :audio_id
+      :audio_id,
+      :title,
+      :slug
     ])
     |> validate_publish_state()
     |> maybe_set_published_at()
+    |> TitleSlug.maybe_generate_slug()
+    |> TitleSlug.unique_constraint()
```

Comment:
Shouldn't that just be scoped to a network?

Reply:
You mean representing an AudioPublication as network+audio in URLs, so we never need the title slug? My thought was that since it's very similar to an episode, it should behave in a similar way.

Reply:
No. Network slug and Audio slug.

Reply:
Audio doesn't have a title/slug 🤔

Reply:
I meant network/slug and audiopublication/slug - sorry.